job-dsls/jobs/scrclr_scan_seed_job.groovy

```diff
@@ -0,0 +1,26 @@
+job('srcclr_scan_seed_job') {
+    description('Scan job, which generates scanning jobs for upstream projects')
+    parameters {
+        stringParam('REPO_FILE_URL','https://raw.githubusercontent.com/kiegroup/droolsjbpm-build-bootstrap/master/script/repository-list.txt','URL of the repository-list.txt file')
+        stringParam('KIE_JENKINS_SCRIPTS_REPO', 'https://github.com/kiegroup/kie-jenkins-scripts', '')
+        stringParam('KIE_JENKINS_SCRIPTS_BRANCH', 'master', '')
+        stringParam('JOB_PATH', '', '')
+    }
+    scm {
+        git {
+            remote {
+                name('origin')
+                url("${KIE_JENKINS_SCRIPTS_REPO}")
+            }
+            branch("${KIE_JENKINS_SCRIPTS_BRANCH}")
+        }
+    }
+
+    steps{
+        shell('curl ${REPO_FILE_URL} -o repository-list.txt')
+        dsl{
+            external('job-dsls/jobs/srcclr_scan_job.groovy','job-dsls/jobs/srcclr_scan_pipeline.groovy')
```

Comment:
Is there a reason to use dsl {}  as opposed to jobDsl {} in already existing seed job?

Reply:
One more idea, is there a reason why this seed job is here? I mean, the existing seed job \[1] could have been used as well (it would immediately create the source clear jobs on Jenkins).

The two remaining scripts would then be self-contained and not fail the tests. If you need to reuse a common functionality, I guess it could be put into a class in \[2] - but I am not very much experienced with it.

\[1] https://github.com/kiegroup/kie-jenkins-scripts/blob/master/job-dsls/jobs/seed_job.groovy
\[2] https://github.com/kiegroup/kie-jenkins-scripts/tree/master/job-dsls/src/main/groovy/org/kie/jenkins/jobdsl

Reply:
As discussed, we want to keep SourceClear scanning as separate from the rest of the process as possible. So, that's why we want to keep a separate seed job for now.